GOVERNANCE-elections.md

```diff
@@ -37,6 +38,40 @@ evolve as the community and project change.
 - [Changes to the charter](#changes-to-the-charter)
 - [Authority, Facilitation, and Decision Making](#authority-facilitation-and-decision-making)
 
+## Values
+
+<!-- This is where you put the core values or principles of your project, like
```

Comment:
@geekygirldawn I just wanted to check that these documents are still using the old method of html comments but that long term they will move over to using a howto guide, right?